examples/src/main/java/io/numaproj/numaflow/examples/function/evenodd/Forward.java

```diff
@@ -1,22 +1,22 @@
 package io.numaproj.numaflow.examples.function.evenodd;
 
+import io.numaproj.numaflow.function.Datum;
 import io.numaproj.numaflow.function.FunctionServer;
 import io.numaproj.numaflow.function.Message;
 import io.numaproj.numaflow.function.map.MapFunc;
-import io.numaproj.numaflow.function.v1.Udfunction;
```

Comment:
Why do we create our own `Datum` class instead of using the one created by protobuf?